test/functional/apps/dashboard/dashboard_state.ts

```diff
@@ -272,7 +272,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
             const allPieSlicesColor = await pieChart.getAllPieSliceStyles('80,000');
             let whitePieSliceCounts = 0;
             allPieSlicesColor.forEach((style) => {
-              if (style.indexOf('rgb(255, 255, 255)') > 0) {
+              if (style.indexOf('rgb(255, 255, 255)') > -1) {
```

Comment:
does this mean that this test was always wrong?

Reply:
No, the difference is that vislib returns `fill: rgb(255, 255, 255)` while es-charts `rgb(255, 255, 255)`

Reply:
ok good to know, thanks